manifests/init.pp

```diff
@@ -79,6 +79,20 @@
 #                        mongo database has slow I/O, then setting a higher number may resolve issues where workers are
 #                        going missing incorrectly.
 #
+# $manage_candlepin::    Whether to manage candlepin
+#
+# $manage_qpid::         Whether to manage qpid
+#
+# $manage_pulp::         Whether to manage pulp. Currently requires $manage_application and $manage_candlepin to be true as well.
+#
+# $manage_application::  Whether to manage application (katello web ui)
```

Comment:
I think I keep getting hung up on application and wonder if something somewhat more specified could work. Ideas that come to mind: server, web_server, rails_server, foreman_server, rails_application, foreman_application

Reply:
I like foreman_application